lib/plug/csrf_protection.ex

```diff
@@ -51,18 +51,21 @@ defmodule Plug.CSRFProtection do
 
   defmodule InvalidCSRFTokenError do
     @moduledoc "Error raised when CSRF token is invalid."
-    message = "invalid CSRF (Cross Site Forgery Protection) token. Make sure that all " <>
-              "your non-HEAD and non-GET requests include the '_csrf_token' as part of form " <>
-              "params or as a value in your request's headers with the key 'x-csrf-token'"
+
+    message =
+      "invalid CSRF (Cross Site Forgery Protection) token, make sure all "
+      <> "requests include a '_csrf_token' param or an 'x-csrf-token' header"
```

Comment:
I think we should make it clear that only non-HEAD and non-GET requests as before. :)